src/components/BookList.js

```diff
@@ -37,7 +46,14 @@ class BookList extends React.Component {
       return (
         <ul className="book-list">
           {books.map((book, i) => (
-            <Book key={book.isbn} index={book.isbn} details={books[i]} />
+            <Book
+              key={book.isbn}
+              isbn={book.isbn}
+              details={books[i]}
+              deleteBook={this.props.deleteBook}
+              loadBook={this.props.loadBook}
+              adminMode={this.props.adminMode}
```

Comment:
Could adminMode take advantage of React.Context?

Reply:
Oooh that does look promising. React.Context wasn't covered in any of my training. I'll file a separate issue to look into parts of state that could live in Context instead.